Random ImagesPipeline test failure

Travis CI sometimes fails builds because of a `ImagesPipeline` test failure.

Example failed build: https://travis-ci.org/scrapy/scrapy/jobs/169472958

```
=================================== FAILURES ===================================
___ ImagesPipelineTestCaseCustomSettings.test_custom_settings_for_subclasses ___
self = <tests.test_pipeline_images.ImagesPipelineTestCaseCustomSettings testMethod=test_custom_settings_for_subclasses>
    def test_custom_settings_for_subclasses(self):
        """
            If there are custom settings for subclass and NO class attributes, pipeline should use custom
            settings.
            """
        class UserDefinedImagePipeline(ImagesPipeline):
            pass

        prefix = UserDefinedImagePipeline.__name__.upper()
        settings = self._generate_fake_settings(prefix=prefix)
        user_pipeline = UserDefinedImagePipeline.from_settings(Settings(settings))
        for pipe_attr, settings_attr in self.img_cls_attribute_names:
            # Values from settings for custom pipeline should be set on pipeline instance.
            custom_value = settings.get(prefix + "_" + settings_attr)
>           self.assertNotEqual(custom_value, self.default_pipeline_settings[pipe_attr])
/home/travis/build/scrapy/scrapy/tests/test_pipeline_images.py:354: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
self = <tests.test_pipeline_images.ImagesPipelineTestCaseCustomSettings testMethod=test_custom_settings_for_subclasses>
first = 90, second = 90, msg = None
    def assertNotEqual(self, first, second, msg=None):
        """
            Fail the test if C{first} == C{second}.

            @param msg: if msg is None, then the failure message will be
            '%r == %r' % (first, second)
            """
        if not first != second:
>           raise self.failureException(msg or '%r == %r' % (first, second))
E           twisted.trial.unittest.FailTest: 90 == 90
/home/travis/build/scrapy/scrapy/.tox/py35/lib/python3.5/site-packages/twisted/trial/_synctest.py:468: FailTest
== 1 failed, 1396 passed, 15 skipped, 12 xfailed, 1 xpassed in 423.24 seconds ==
ERROR: InvocationError: '/home/travis/build/scrapy/scrapy/.tox/py35/bin/py.test --cov=scrapy --cov-report= scrapy tests'
___________________________________ summary ____________________________________
ERROR:   py35: commands failed
```

--- a/tests/test_pipeline_images.py
+++ b/tests/test_pipeline_images.py
@@ -244,7 +244,7 @@ class ImagesPipelineTestCaseCustomSettings(unittest.TestCase):
             return "".join([chr(random.randint(97, 123)) for _ in range(10)])
 
         settings = {
-            "IMAGES_EXPIRES": random.randint(1, 1000),
+            "IMAGES_EXPIRES": random.randint(100, 1000),
             "IMAGES_STORE": self.tempdir,
             "IMAGES_RESULT_FIELD": random_string(),
             "IMAGES_URLS_FIELD": random_string(),

